### PR TITLE
Magento-cloud-components: Release 1.0.5

### PR DESCRIPTION
**Pull request by YPyltiai**
## Purpose of this pull request

Added release notes for Magento Cloud Components v1.0.5 release.

## Affected DevDocs pages

https://devdocs.magento.com/cloud/release-notes/mcc-release-notes.html

whatsnew
Added [Magento Cloud Components v1.0.5 release notes](https://devdocs.magento.com/cloud/release-notes/mcc-release-notes.html).

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
@YPyltiai thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/cloud/release-notes/mcc-release-notes.md [hunk 1]
```diff
@@ -11,8 +11,13 @@ The [Magento Cloud Components](https://github.com/magento/magento-cloud-componen
 
 The `{{site.data.var.mcc-package}}` package uses the following version sequence: `<major>.<minor>.<patch>`.
 
+## v1.0.5
+*Release date: June 26, 2020*<br/>
+
+-  {:.fix}Fixed an issue introduced in package version 1.0.4 with cache flush operation through Fastly extension failing during the deploy phase, causing the deployment to be interrupted.
```
Comment (meker12) [inline src/cloud/release-notes/mcc-release-notes.md:17]:
```suggestion
-  {:.fix}Fixed an issue introduced in {{site.data.var.mcc-package}} version 1.0.4 that caused the flush cache operation initiated by the Fastly extension to fail during the deploy phase, which interrupts deployment.
```

Comment (bbatsche) [inline src/cloud/release-notes/mcc-release-notes.md]:
```suggestion
-  {:.fix}Fixed an issue introduced in package version 1.0.4 with cache flush operation failing during the deploy phase, causing the deployment to be interrupted.
```

File: src/cloud/release-notes/mcc-release-notes.md [hunk 2]
```diff
@@ -11,8 +11,13 @@ The [Magento Cloud Components](https://github.com/magento/magento-cloud-componen
 
 The `{{site.data.var.mcc-package}}` package uses the following version sequence: `<major>.<minor>.<patch>`.
 
+## v1.0.5
```
Comment (meker12) [inline src/cloud/release-notes/mcc-release-notes.md:14]: @bbatsche  Is this version OK?

File: src/cloud/release-notes/mcc-release-notes.md [hunk 3]
```diff
@@ -11,8 +11,13 @@ The [Magento Cloud Components](https://github.com/magento/magento-cloud-componen
...
+*Release date: June 26, 2020*<br/>
+
+-  {:.fix}Fixed an issue introduced in {{site.data.var.mcc-package}} version 1.0.4 that caused the flush cache operation initiated by the Fastly extension to fail during the deploy phase, which interrupts deployment.
```
Comment (meker12) [inline src/cloud/release-notes/mcc-release-notes.md:17]:
```suggestion
-  {:.fix}Fixed an issue introduced in {{site.data.var.mcc-package}} version 1.0.4 that caused the flush cache operation to fail during the deploy phase, which interrupts deployment.
```

Comment (hguthrie) [inline src/cloud/release-notes/mcc-release-notes.md:17]: The last clause indicates the the deploy phase interrupts deployment. perhaps a little fix...

```suggestion
-  {:.fix}Fixed an issue introduced in {{site.data.var.mcc-package}} version 1.0.4 that caused the flush cache operation initiated by the Fastly extension to fail during the deploy phase, interrupting the deployment process.
```

**Comment by YPyltiai** [comment]
@meker12 , @hguthrie , @bbatsche - merged all change requests. If looks good - please squash & merge. Thanks!

**Comment by meker12** [comment]
running tests

**Pull request merged**